plugins/azure_ad_admin/plugin.spec.yaml

```diff
@@ -23,6 +23,302 @@ hub_tags:
+      title: On Premises Security Identifier
+      type: string
+      description: On Premises Security Identifier
+      required: false
+    onPremisesSamAccountName:
+      title: On Premises Sam Account Name
+      type: string
+      description: On Premises Sam Account Name
+      required: false
+    onPremisesSyncEnabled:
+      title: On Premises Sync Enabled
+      type: string
+      description: On Premises Sync Enabled
+      required: false
+    onPremisesUserPrincipalName:
+      title: On Premises User Principal Name
+      type: string
+      description: On Premises User Principal Name
+      required: false
+    otherMails:
+      title: Other Mails
+      type: "[]string"
+      description: Other Mails
+      required: false
+    passwordPolicies:
+      title: Password Policies
+      type: string
+      description: Password Policies
+      required: false
+    officeLocation:
+      title: Office Location
+      type: string
+      description: Office Location
+      required: false
+    postalCode:
+      title: Postal Code
+      type: string
+      description: Postal Code
+      required: false
+    preferredDataLocation:
+      title: Preferred Data Location
+      type: string
+      description: Preferred Data Location
+      required: false
+    proxyAddresses:
+      title: Proxy Addresses
+      type: array
+      description: Proxy Addresses
+      required: false
+    refreshTokensValidFromDateTime:
+      title: Refresh Tokens Valid From Date Time
+      type: date
+      description: Refresh Tokens Valid From Date Time
+      required: false
+    imAddresses:
+      title: Im Addresses
+      type: array
+      description: Im Addresses
+      required: false
+    isResourceAccount:
+      title: Is Resource Account
+      type: string
+      description: Is Resource Account
+      required: false
+    showInAddressList:
+      title: Show In Address List
+      type: string
+      description: Show In Address List
+      required: false
+    signInSessionsValidFromDateTime:
+      title: Sign In Sessions Valid From Date Time
+      type: date
+      description: Sign In Sessions Valid From Date Time
+      required: false
+    state:
+      title: State
+      type: string
+      description: State
+      required: false
+    streetAddress:
+      title: Street Address
+      type: string
+      description: Street Address
+      required: false
+    surname:
+      title: Surname
+      type: string
+      description: Surname
+      required: false
+    usageLocation:
+      title: Usage Location
+      type: string
+      description: Usage Location
+      required: false
+    userPrincipalName:
+      title: User Principal Name
+      type: string
+      description: User Principal Name
+      required: false
+    userType:
+      title: User Type
+      type: string
+      description: User Type
+      required: false
+    authorizationInfo:
+      title: Authorization Info
+      type: object
```

Comment:
Can object structure be identified and a custom type added for general object types?